common/djangoapps/student/views.py

```diff
@@ -814,8 +814,9 @@ def dashboard(request):
 
     if bundles_on_dashboard_flag.is_enabled():
         programs_data = meter.programs
-        if programs_data:
-            meter.programs = [get_programs(request.site, uuid=programs_data[0]['uuid'])]
+        if programs_data and inverted_programs:
+            program_uuid = inverted_programs.values()[0][0]['uuid']
```

Comment:
getting uuid of first related program, not first of all programs

Reply:
(think this might be at least one of the issues. not visible locally because i only had one program, which was also coming back first in the list of all programs)

Reply:
just making sure, is there any case where `inverted_programs.values()` will be an empty array (of course, given `programs_data and inverted_programs is true`)?

Reply:
pretty sure there isn't, but i added a check to the if statement just in case

Reply:
ah sorry, I forgot that empty arrays are falsy -- good point